content/docs/command-reference/plots/modify.md

```diff
@@ -123,6 +123,29 @@ plots:
       title: Accuracy
 ```
 
+## Example: Make a regular output into a plot
+
+We can use `dvc plots modify` to make an <abbr>output</abbr> or metrics file of
+a stage into a plot. Lets take an example stage `evaluate` which has an
+<abbr>output</abbr> `logs.csv`:
+
+```yaml
+evaluate:
+  cmd: python src/evaluate.py
+    deps:
+    - src/evaluate.py
+    outs:
+    - logs.csv
+    plots:
+    - scores.json
+```
```

Comment:
Let's mention somewhere that this is a `dvc.yaml` file (so its linked to the DVC files page)